calico/felix/plugins/fiptgenerator.py

```diff
@@ -488,7 +488,18 @@ def filter_forward_chain(self, ip_version):
                 (CHAIN_FORWARD, iface_match),
             ])
 
-        return forward_chain, set([CHAIN_FROM_ENDPOINT, CHAIN_TO_ENDPOINT])
+        # If we get to this point in the chain, the packet is not from or to
+        # a workload endpoint.  Hence, it must be being forwarded through
+        # this host, potentially via hst endpoints that we want to police.
+        # Send the traffic to the host endpoint chains for processing.
+        # If there are no host endpoints configured, these chains are no-ops.
+        forward_chain.append("--append {chain} --jump {jump}".format(
```

Comment:
We normally --goto the FROM-IFACE and TO-IFACE chains, rather than --jumping to them, and I think they do use RETURN - so care is needed about whether that will behave as desired.

What's your plan for testing this?  It would be better to have a successful ST before merging.